bluemira/base/parameter_frame/_parameter.py

```diff
@@ -56,7 +56,7 @@ def __init__(
         self,
         name: str,
         value: ParameterValueType,
-        unit: str = "",
+        unit: Union[str, pint.Unit] = pint.Unit("dimensionless"),
```

Comment:
I'm beginning to think we shouldn't be allowing `pint` types as inputs. If we do ever want to manage units differently, we'll be making breaking API changes if we move away from `pint`.

It may never be the case that we stop using `pint`, but I feel for something like this it's better practice to encapsulate the way we're managing units. What do you think? Maybe I'm missing an advantage to allowing `pint` types?

Reply:
I havent got a good argument to the contrary so done